ledger/complete/wal/syncrename.go

```diff
@@ -53,6 +53,13 @@ func (s *SyncOnCloseRenameFile) Close() error {
 		return fmt.Errorf("cannot sync file %s: %w", s.file.Name(), err)
 	}
 
+	// s.file.Sync() was already called, so we pass fsync=false
+	err = evictFileFromLinuxPageCache(s.file, false, s.logger)
+	if err != nil && s.logger != nil {
```

Comment:
```suggestion
	if err != nil  {
```

I don't think we need to handle the `logger == nil` case. The logger should always be available, if not, it's a bug. 

Same in other places.

Reply:
Thanks for the suggestion! :+1:   Done in commit dd7d9551f630bda3c693aa0e9f931b64d44dae6a.